Studio-Lovelies/GG-JointJustice-Unity
# Investigation: Track already examined choices

This PR is based on #481 which should be integrated first.

## Summary
Adds a placeholder checkmark texture to choices already examined inside investigations:
![image](https://github.com/user-attachments/assets/bc9412dc-2515-46e2-ad96-fb1e07d3e8e3)

## Before/after screenshots and/or animated gif

https://github.com/user-attachments/assets/891b59ca-782a-4e31-9986-bb19c869f2a9

## Testing instructions
1. Open the `Game` scene
2. Set the `GameStarter` to `InvestigationUI`
3. Select any `Talk` or `Move` choice
4. Open the sub menu again and notice your choice being prefixed with a checkmark

## Additional information

- `[x]` Changes UI
- `[ ]` Introduces new feature
- `[ ]` Removes existing feature
- `[x]` Has associated resource: 
  - #456
